projects/collections/collection.js

```diff
@@ -3,10 +3,37 @@ Projects = new Mongo.Collection('projects');
 //Add search index with easy search
 ProjectsIndex  = new EasySearch.Index({
     collection: Projects,
-    fields: ['name','tags','postalAdress'],
-    engine: new EasySearch.Minimongo()
-  });
-  
+    fields: ['name','tags'],
+    engine: new EasySearch.Minimongo({
+      selector: function (searchObject, options, aggregation) {
+        // Set selector to Default
+        let selector = this.defaultConfiguration().selector(searchObject, options, aggregation);
+
+        // Reactively gets the queryparams to sort the /projects page
+        let category = FlowRouter.getQueryParam("cat");
```

Comment:
Lets go ahead and use the full words `category` and `country` here. This makes the URL arguments explicit and slightly more semantic.